File: CPP/ParkingLot.cpp
#include <string>

using namespace std;

class ParkingLot {
    private:
        int lot_capacity;
        int occupied;
        
    public:
        ParkingLot() {
            this->lot_capacity = -1;
            this->occupied = -1;
        }
        ParkingLot(int lot_capacity, int occupied){
            this->lot_capacity = lot_capacity;
            this->occupied = occupied;
        }

        int getLotCapacity() { return this->lot_capacity; }
        int getOccupied() { return this->occupied; }

        void setLotCapacity(int lot_capacity) { this->lot_capacity = lot_capacity; }
        void setOccupied(int occupied) { this->occupied = occupied; }
};

File: Python/ParkingLot.py
class ParkingLot:
    # Attribute Declaration
    __lot_capacity = -1;
    __occupied = -1;


    # Constructor
    def __init__(self, lot_capacity=-1, occupied=-1):
        self.__lot_capacity = lot_capacity
        self.__occupied = occupied = occupied

    # Getter-Setter
    # Get Methods
    def getLotCapacity(self):
        return self.__lot_capacity
    def getOccupied(self):
        return self.__occupied

    # Set Methods
    def setLotCapacity(self, lot_capacity):
        self.__lot_capacity = lot_capacity
    def setOccupied(self, occupied):
        self.__occupied = occupied